demos/tests/run_tests.py

```diff
@@ -51,12 +51,15 @@
 from cases import Demo
 from data_sequences import DATA_SEQUENCES
 
+import shutil
+import requests
+
 scopes = {
     'base': importlib.import_module('cases').DEMOS,
     'performance': importlib.import_module('performance_cases').DEMOS,
 }
 COCO128_URL = "https://ultralytics.com/assets/coco128.zip"
-
+VIDEO_URL = "https://storage.openvinotoolkit.org/data/test_data/videos/head-pose-face-detection-male.mp4"
```

Comment:
This is the same video used in the `gaze_estimation_demo` readme file ([here](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/gaze_estimation_demo/cpp_gapi))
<img width="885" alt="image" src="https://github.com/openvinotoolkit/open_model_zoo/assets/52502732/4d4f7850-ed46-4a64-a7ed-eb455d4dfbee">

<img width="419" alt="image" src="https://github.com/openvinotoolkit/open_model_zoo/assets/52502732/b69e4c16-3798-4fe0-ad4a-1a35ce16e030">